calico/felix/fetcd.py

```diff
@@ -151,6 +156,7 @@ def _on_actor_started(self):
         self._resync_greenlet.start()
         self.status_reporter.start()
         self._status_reporting_greenlet.start()
+        self._usage_count_greenlet.start()
```

Comment:
The start should be conditional on the config too.  Think we've lost that somewhere.